apps/omg_watcher/lib/omg_watcher/api/account.ex

```diff
@@ -18,11 +18,13 @@ defmodule OMG.Watcher.API.Account do
   """
 
   alias OMG.Watcher.DB
+  use Appsignal.Instrumentation.Decorators
 
   @doc """
   Returns a list of amounts of currencies that a given address owns
   """
   @spec get_balance(OMG.Crypto.address_t()) :: list(DB.TxOutput.balance())
+  @decorate transaction(:Watcher_API_Account)
```

Comment:
no snakel case + see other comments on naming conventions. In this particular case I think the default `web` namespace is totally ok.

Oh, actually, you don't need a transaction at all - Phoenix does this already!

Find & fix all occurrences

Reply:
Why is this a Transaction and others are transaction events? If I understood the docs correctly you need to start a transaction first, which is followed by transaction events...

Reply:
what others do you mean? 

I guess we should use `event` whenever we somehow know, that a transaction is "ongoing", and this PR attempts to do that, with some polishes required (more on such suggestions in my comments).

Here I think (and in all other APIs called directly by Phoenix controllers), we can safely assume that we're running under phoenix (or some other "entrypoint") and the txs are already ongoing. 

OTOH, I'm not sure whether we'd like to have an `event` here, instead of just nothing. I suggested "nothing", but come to think of it - an `event` would allow to see the portion of the request handling spent in here, as contrasted with Phoenix overhead. Not sure which is better - we can for now go with a "rich" version (with `event`) and later ditch the events, if they're noisy or overheady.

Reply:
It seems like the flow of what a transaction is and what an event is somewhat hidden in the macros. implicit implementation means we will have difficulties measuring the performance impact and because of that, I really think we ought to consider other approaches